prime-router/src/scripts/db_one_off/1974_customer_status.sql

```diff
@@ -0,0 +1,28 @@
+
+/*
+    This will need to be run once in each environment, but is safe to re-run if needed.
+    Backfills all customerStatus values with correct status as of 9/20/2021.  It will create the missing property of the object if not present, and then insert the value specified.
+ */
+
+-- set all customerStatus to default 'inactive'
+update setting
+set values = jsonb_set(values, '{customerStatus}', '"inactive"', true)
+where type in ('SENDER','RECEIVER');
+
+-- set active senders and receivers to 'active'
+update setting
+set values = jsonb_set(values, '{customerStatus}', '"active"', true)
+where type in ('SENDER','RECEIVER')
+  and  values ->> 'organizationName' in ('simple_report','waters','safehealth','cue','inbios','imagemover','anavasidx','strac',
```

Comment:
Feel free to make minor last minute additions to this without having to do another PR.